src/nuget.config

```diff
@@ -1,7 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Oh dear. Does this mean all our repos now have this redundant file hanging around in `src/`, as well as the intended one in the root?

Reply:
Yes it does. RepoStandards sync doesn't have a way to remove a file.